### PR TITLE
Add permalink metedata example

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-489cc5024c/how-to/)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Follow these steps to create a new article within a product folder of the **rack
 4. Enter header information using the format shown in the following example:
 
            ---
+           permalink: Title-of-article
```
Comment (DonNateR) [inline CONTRIBUTING.md]: Make "Title" lowercase. Add `/` to the end of the line.

**Comment by kylelaffoon** [comment]
@DonNateR Thanks. Correction applied.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25df908f99/how-to/)

**Pull request merged**